spreg/sur.py

```diff
@@ -676,34 +695,34 @@ class ThreeSLS(BaseThreeSLS, REGI.Regimes_Frame):
     dictionaries for Y and X, and sur_dictZ for endogenous variables (yend) and
     additional instruments (q).
 
-    >>> bigy,bigX,bigyvars,bigXvars = pysal.spreg.sur_utils.sur_dictxy(db,y_var,x_var)
-    >>> bigyend,bigyendvars = pysal.spreg.sur_utils.sur_dictZ(db,yend_var)
-    >>> bigq,bigqvars = pysal.spreg.sur_utils.sur_dictZ(db,q_var)
+    >>> bigy,bigX,bigyvars,bigXvars = spreg.sur_dictxy(db,y_var,x_var)
+    >>> bigyend,bigyendvars = spreg.sur_dictZ(db,yend_var)
+    >>> bigq,bigqvars = spreg.sur_dictZ(db,q_var)
 
     We can now run the regression and then have a summary of the output by typing:
     print(reg.summary)
 
     Alternatively, we can just check the betas and standard errors, asymptotic t
     and p-value of the parameters:
 
-    >>> reg = ThreeSLS(bigy,bigX,bigyend,bigq,name_bigy=bigyvars,name_bigX=bigXvars,name_bigyend=bigyendvars,name_bigq=bigqvars,name_ds="NAT")
+    >>> reg = spreg.ThreeSLS(bigy,bigX,bigyend,bigq,name_bigy=bigyvars,name_bigX=bigXvars,name_bigyend=bigyendvars,name_bigq=bigqvars,name_ds="NAT")
```

Comment:
It may be a good time to consider adopting [`black`](https://black.readthedocs.io/en/stable/) for code formatting for this (and other) `pysal` submodules.

```python
reg = spreg.ThreeSLS(
    bigy,
    bigX,
    bigyend,
    bigq,
    name_bigy=bigyvars,
    name_bigX=bigXvars,
    name_bigyend=bigyendvars,
    name_bigq=bigqvars,
    name_ds="NAT"
)
````

Reply:
Not a big fan of `black` - makes code too long and possibly harder to read. 

You may open another PR specifically for this afterward if you'd like.